src/Controls/PlanarControls.js

```diff
@@ -82,7 +83,7 @@ const defaultOptions = {
     rotateSpeed: 2.0,
     minPanSpeed: 0.05,
     maxPanSpeed: 15,
-    zoomTravelTime: 0.2,
+    zoomTravelTime: 0.2,  // must be a number
```

Comment:
why you this comment?

Reply:
I added this comment so that the default value can't be changed to `auto` (which is a relevant value for `zoomTravelTime` with a perspective camera).
Was this value to be switched to `'auto'`, the call of `initiateZoom()` shall trigger an error : it would set const `travelDuration` to `'auto'`, then try to divide a number by `travelDuration`.
This comment is a warning so that the default value is not changed to `'auto'` in the future.